### ISSUE TITLE
can't use tabnine

### ISSUE THREAD

**Issue by Zxrcoming** [neoclide/coc-tabnine#11]
when I type tabnine.openConfig
it shows:
```vim
[coc.nvim] error: Uncaught exception: Error: spawn /home/zxr/.config/coc/extensions/coc-tabnine-data/binaries/2.0.4/x86_64-unknown-linux-gnu/TabNi
ne EACCES
[coc.nvim] error:     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
[coc.nvim] error:     at onErrorNT (internal/child_process.js:415:16)
[coc.nvim] error:     at process._tickCallback (internal/process/next_tick.js:63:19)
[coc.nvim] Command error: Cannot read property 'on' of undefined
```
and the size of the tabnin's binary is only 1.7MB.
here is my `cocinfo` :
```vim
## versions

vim version: vim - vi improved 8.1 (2018 may 18, compiled jul 31 2019 10:22:31)
node version: v10.16.0
coc.nvim version: 0.0.73-baa3b1f169
term: dumb
platform: linux

## messages
messages maintainer: bram moolenaar <bram@vim.org>
"~/.vim/vimrc" 354l, 8389c
[coc.nvim] error: uncaught exception: error: spawn /home/zxr/.config/coc/extensions/coc-tabnine-data/binaries/2.0.4/x86_64-unknown-linux-gnu/TabNine EACCES
[coc.nvim] error:     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
[coc.nvim] error:     at onErrorNT (internal/child_process.js:415:16)
[coc.nvim] error:     at process._tickCallback (internal/process/next_tick.js:63:19)
[coc.nvim] Command error: Cannot read property 'on' of undefined
```

**Comment by chemzqm** [comment]
Broken download, remove it and download again.

**Comment by Zxrcoming** [comment]
still not working ……

**Comment by chemzqm** [comment]
The download uri was changed, fixed on 1.2.0

**Comment by 191205** [comment]
I'm facing the same problem and message, I usually get timeout error
https://imgur.com/gtT8Uhv

You can see I removed and reinstall it but no luck
